products/windowsServer.md

```diff
@@ -50,6 +50,13 @@ releases:
     latest: 10.0.18363
     link: https://learn.microsoft.com/lifecycle/announcements/windows-server-1909-end-of-servicing
 
+-   releaseCycle: "1903"
+    releaseDate: 2019-05-21
+    eoas: 2020-12-08
+    eol: 2020-12-08
+    latest: 10.0.18362
+    link: https://learn.microsoft.com/windows/release-health/windows-server-release-info
```

Comment:
https://learn.microsoft.com/windows/release-health/windows-server-release-info doesn't mention 1903, this one does:

```suggestion
    link: https://learn.microsoft.com/en-us/lifecycle/products/windows-server-semiannual-channel
```